src/components/Transcript/TranscriptSection.ts

```diff
@@ -88,23 +89,45 @@ export class TranscriptSection implements RedomComponent {
     }
   };
 
+  private createSpeakerElements = () => {
+    const { speaker } = this.paragraph;
+    const firstname = speaker.unknown ? 'Mluvčí' : `${speaker.firstname ?? ''}` as string;
+    const surname = speaker.unknown ? '' : speaker.surname as string;
+    const role = speaker.unknown || speaker.role === undefined ? '' : speaker.role as string;
+    const speakerElements = [
+      { name: 'firstname', className: '', text: firstname },
+      { name: 'surname', className: '', text: surname },
+      { name: 'role', className: '', text: role },
+    ];
+    this.paragraph.speakerKeywordInstances.forEach((instance) => {
+      const className = extractKeywordsClassNames(
+        SPEAKER_KW_PREFIX,
+        [instance],
+      );
+      const { mentions } = instance.keyword;
```

Comment:
Why is there a loop over all mentions when all the mentions are already accounted for in the keyword instances?